Remove `additionalProperties` from options for primitives

daedalus28 opened the pull request:
Closes #33 

Removes `additionalProperties` from options for primitives.

I didn't see a contributor guide, so let me know if there are any tests I should be running or authoring. I'm also not sure how this project handles releases so I didn't bump the package.json.

m1212e commented:
Thanks, we use github releases and inject the version during the build so no worries I'll take care of that! Also: Tests would be very nice to have but unfortunately I did not yet have time/motivation to add them.

daedalus28 commented:
I fixed some linting issues the ci workflow idenitified, although at least one of them is not in code that I changed. Let me know if there's anything else you need me to do on this one.

m1212e commented:
Thank you very much!
